ide-migration/server/migration/api/track-service.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2021 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+let dao = require('db/v4/dao');
+let database = require('db/v4/database');
+let migrationsTable = null;
+
+
+class TrackService {
+    setupTable() {
+        migrationsTable = dao.create({
+            table: "MIGRATIONS",
+            properties: [{
+                name: "id",
+                column: "ID",
+                type: "BIGINT",
+                id: true
+            }, {
+                name: "deliveryUnit",
+                column: "DELIVERY_UNIT",
+                type: "VARCHAR",
+                required: false
+            }, {
+                name: "startedOn",
+                column: "STARTED_ON",
+                type: "TIMESTAMP",
+                required: false
+            }, {
+                name: "lastUpdated",
+                column: "LAST_UPDATED",
+                type: "TIMESTAMP",
+                required: false
+            },
+                {
+                    name: "status",
+                    column: "STATUS",
+                    type: "VARCHAR",
+                    required: false
+                }]
+
+        });
```

Comment:
The `MIGRATION` table should be created in the `local` datasource type under the `SystemDB`. Is this still possible with the `DAO` API?